src/chapters/sql-part2/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Library
+===============
+
+Word has gotten out about your amazing SQL skills!
+The head librarian at the local library asked if you could help with their database.
+
+Database Setup
+--------------
+
+Let's start by creating a new schema in MySQL Workbench called ``library``.
+Once you have accepted all defaults and hit "Apply", run the provided scripts to create tables and add data.
+By the end of your setup, you should have 6 tables:
+
+#. ``book``
+#. ``author``
+#. ``patron``
+#. ``reference_book``
+#. ``genre`` 
+#. ``loan``
+
+Loan Out a Book
+---------------
+
+A big function that you need to implement for the library is a query that updates the database when a book is loaned out.
+
+The query needs to perform the following functions:
+
+#. Change ``available`` to ``FALSE`` for the appropriate book.
+#. Add a new row to the ``loan`` table with today's date as the date out and the ids in the row need to match the appropriate ``patron_id`` and ``book_id``.
+#. Update the appropriate ``patron`` with the ``loan_id`` for the new row created in the ``loan`` table.
+
+You can use any patron and book that strikes your fancy to create this query!
+
+Check a Book Back In
+--------------------
+
+Working with the same patron and book, create the new query!
+
+The other key function that we need to implement is checking a book back in.
+To do so, the query needs to:
```

Comment:
Similar to my earlier comment - Is this going to be a single query or a sequence of queries?

Reply:
Either way, changed "query" to "script" so that people can accomplish the task however!